mlrun/feature_store/api.py

```diff
@@ -179,6 +182,9 @@ def ingest(
     :param spark_context: local spark session or True to create one, example for creating the spark context:
                           `spark = SparkSession.builder.appName("Spark function").getOrCreate()`
     :param transformer:   custom transformation function which accepts dataframe and **kwargs as input
+    :param filter_start   datetime, low limit of time needed to be filtered. format '2020-12-01 17:33:15'
+    :param filter_end     datetime, high limit of time needed to be filtered. format '2020-12-01 17:33:15'
```

Comment:
you're writing datetime but according to the format in the end and your tests I see you're expecting string here
I think we should support both datetime and string here, but if one of them - then datetime
Note that the format you're using is missing the timezone info.
Lastly, I'm not in love with this naming,
what about `start_time` `end_time` and `time_column` ? 
@yaronha WDYT ?

Reply:
IMO we should be specific and just have start/end time (and the time_key as the column) vs defining generic filter